src/core/cursor.ts

```diff
@@ -106,7 +106,7 @@ export const showCursor = () => {
 
   cursorRequestedToBeHidden = false
   cursorEl.style.display = 'flex'
-  cursorline.style.display = ''
+  cursorline.style.display = 'none'
```

Comment:
This should probably be configurable?

Reply:
Not sure; neovim provides its own cursorline